mqtt.md

```diff
@@ -863,6 +979,47 @@ Gets the LAN access code of the printer
 }
 ```
 
+## system.set_accessories.nozzle
+
+Set the type and diameter of the printer nozzle
+
+**Request**
+```json
+{
+    "system": {
+        "sequence_id": "2047",
+        "command": "set_accessories",
+        "accessory_type": "nozzle",
+        "nozzle_type": "stainless_steel", // "stainless_steel" or "hardened_steel"
+        "nozzle_diameter": "0.4" // "stainless_steel": "0.2"/"0.4" or "hardened_steel": "0.4"/"0.6"/"0.8"
+    }
+}
+```
+
+**Report**
+
+TODO
+
+## system.set_accessories.upgrade_kit
+
+Set the Printer P1P Official Enclosure Kit
+
+**Request**
+```json
+{
+    "system": {
+        "sequence_id": "2056",
+        "command": "set_accessories",
+        "accessory_type": "upgrade_kit",
+        "upgrade_kit": false // false or true
```

Comment:
I don't think that needs to be commented, as that is natural for a boolean value?